analyzers/tests/SonarAnalyzer.UnitTest/Common/SecurityHotspotTest.cs

```diff
@@ -87,14 +87,16 @@ analyzerName switch
                 "ExecutingSqlQueries" => "ExecutingSqlQueries_NetCore",
                 "UsingCookies" => "UsingCookies_NetCore",
                 "LooseFilePermissions" => "LooseFilePermissions.Unix",
+                "PermissiveCors" => "PermissiveCors.Net",
 #endif
                 _ => analyzerName
             };
 
         private static bool IsTestValid(string analyzerName)
         {
 #if NETFRAMEWORK
-            return analyzerName != nameof(DisablingCsrfProtection);
+            return analyzerName != nameof(DisablingCsrfProtection)
```

Comment:
given that `ASP.NET MVC` is part of the spec, shouldn't there be a `ASP.NET MVC` test case for .NET Framework, and  `ASP.NET Core MVC` for Netcore? basically splitting  the existing test case in two?

or isn't this already the case for `PermissiveCors.WebApi.cs`? shouldn't that be in this test as well? the same as we do for `LooseFilePermissions`?

Reply:
Most of the test cases are common between `ASP.NET MVC` and `Web Api` since the API is defined in a NuGet package. I will rename the file to `PermissiveCors.NetFramework.cs` to reflect that. I will also add more test cases